UI/EspionageSupport.lua

```diff
@@ -129,7 +129,7 @@ function hasDistrict(city:table, districtType:string)
     local hasDistrict:boolean = false;
     local cityDistricts:table = city:GetDistricts();
     for i, district in cityDistricts:Members() do
-        if district:IsComplete() then
+        if district:IsComplete() and not district:IsPillaged() then
```

Comment:
--ARISTOS: to only show available and valid targets in each city, both for espionage overview and selector